swapserverrpc/server.proto

```diff
@@ -677,4 +686,37 @@ message ServerAddressParameters {
 
     // The CSV expiry for the MuSig2 static address output.
     uint32 expiry = 2;
+}
+message ServerWithdrawRequest {
+    // The deposit outpoints the client wishes to close.
+    repeated ServerOutPoint outpoints = 1;
+
+    // The nonces that the client used to generate the coop close tx sigs.
+    repeated bytes client_nonces = 2;
+
+    // The address that the client wants to sweep the static address deposits
+    // to.
+    string client_sweep_addr = 3;
+
+    // The fee rate in sat/kw that the client wants to use for the sweep.
+    uint64 musig_tx_fee_rate = 4;
```

Comment:
I suggest to rename to `musig2_tx_fee_rate` not to imply that MuSig1 protocol is used. Or just `tx_fee_rate`.

There is another instance of a field with name `musig_tx_fee_rate` in `instantout.proto`. I suggest to rename here as well.